### PR TITLE
stratis_cli_cert: add SysfsMonitor class

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #229

### CODE REVIEW

File: testlib/infra.py [hunk 1]
```diff
@@ -169,6 +169,36 @@ def setUp(self):
...
 
 
+class SysfsMonitor(unittest.TestCase):
+    """
+    Manage verification of sysfs files for devices.
+    """
+
+    def tearDown(self):
+        if SysfsMonitor.verify_sysfs:  # pylint: disable=no-member
+            dm_devices = {}
```
Comment (mulkieran) [inline testlib/infra.py:179]:
```
dev_mapper = "/dev/mapper"
dm_devices = {(os.path.basename(os.path.realpath(os.path.join(dev_mapper,  dmdev))), dmdev) for dmdev in os.listdir(dev_mapper_path)}
```

**Review by mulkieran — CHANGES_REQUESTED** [review]
One refactoring request...should do exactly the same thing but with fewer local variables.

File: testlib/infra.py [hunk 2]
```diff
@@ -169,6 +169,36 @@ def setUp(self):
...
+                misaligned_devices = []
+                for dev in os.listdir("/sys/class/block"):
+                    if fnmatch.fnmatch(dev, "dm-*"):
+                        dev_sysfspath = os.path.join(
+                            "/sys/class/block", dev, "alignment_offset"
+                        )
+                        with open(dev_sysfspath, "r", encoding="utf-8") as dev_sysfs:
+                            dev_align = dev_sysfs.read().rstrip()
+                            if int(dev_align) != 0:
+                                misaligned_devices.append(dm_devices[dev])
```
Comment (mulkieran) [inline testlib/infra.py:195]: Try making the elements of the misaligned devices list a formatted string for now, like: ```f"Stratis name: {dm_devices[dev]}, DM name: ..."``` and possibly the non-zero alignment as well, whatever it is.

**Review by mulkieran — CHANGES_REQUESTED** [review]
One more request, then I think this is done.

File: testlib/infra.py (hunk 1 above)
Comment (bgurney-rh) [reply testlib/infra.py]: Resolved (after some corrections)

File: testlib/infra.py (hunk 2 above)
Comment (bgurney-rh) [reply testlib/infra.py]: Resolved.

**Comment by bgurney-rh** [comment]
Squashed into a single commit.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Add ```--verify-sysfs``` argument parsing and related code to stratisd_cert.py as well, and then this is complete.

**Review by mulkieran — APPROVED** [review]

**Comment by bgurney-rh** [comment]
Squashed into a single commit, and reworded commit message.

**Pull request merged**